### PR TITLE
add idelta function documentation

### PR DESCRIPTION
**Pull request by brancz**
Documentation for recently introduced function `idelta`. (see https://github.com/prometheus/prometheus/pull/1875)

### CODE REVIEW

File: content/docs/querying/functions.md [hunk 1]
```diff
@@ -162,6 +162,16 @@ more trends in the data is considered. Both `sf` and `tf` must be between 0 and
 
 `holt_winters` should only be used with gauges.
 
+## `idelta()`
+
+`idelta(v range-vector)`
+
+`idelta(v range-vector)` calculates the difference between the second-to-last
+and last value of each time series element in a range vector `v`, returning an
+instant vector with the given deltas and equivalent labels.
```
Comment (fabxc) [inline content/docs/querying/functions.md]: Maybe "difference between the last two samples in the range vector" is easier to understand?

Comment (brancz) [inline content/docs/querying/functions.md]: rereview @fabxc :)

**Comment by fabxc** [comment]
👍 could you reopen this PR against the `next-release` branch though?

**Comment by brancz** [comment]
no problem @fabxc

**Comment by brancz** [comment]
ignore

**Pull request closed without merging**